### PR TITLE
Hide the picker in "edit all" or clipboard mode

### PR DESCRIPTION
**Pull request by aschempp**
I noticed we sometimes hide the picker radio/checkbox on select-all, but sometimes don't. For tree mode, the layout is broken when both picker and select-all are active.

I think it would also be wise to hide the picker field if something is in the clipboard, because you don't want multiple operations available at the same time. This would also become confusing/problematic with the upcoming operations submenu.

Related to https://github.com/contao/contao/pull/7593

### CODE REVIEW

**Review by Toflar — DISMISSED** [review]

File: core-bundle/contao/classes/DataContainer.php [hunk 1]
```diff
@@ -1042,7 +1042,16 @@ public function initPicker(PickerInterface $picker)
 	{
 		$provider = $picker->getCurrentProvider();
 
-		if (!$provider instanceof DcaPickerProviderInterface || $provider->getDcaTable($picker->getConfig()) != $this->strTable)
+		if (!$provider instanceof DcaPickerProviderInterface || $provider->getDcaTable($picker->getConfig()) != $this->strTable || Input::get('act') == 'select')
```
Comment (leofeyer) [inline core-bundle/contao/classes/DataContainer.php]: The early return here prevents the tree from showing the selected nodes.

### Regular mode



### Clicked on "edit multiple"

Comment (aschempp) [reply core-bundle/contao/classes/DataContainer.php]: I think that is intentional. We now display _select all_ **OR** _paste_ **OR** _the picker_. You won't loose that value if you paste first.

Comment (aschempp) [reply core-bundle/contao/classes/DataContainer.php]: fixed in 8ec639c

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Pull request merged**